docs/development.md

```diff
@@ -244,6 +244,29 @@ redirect_from:
 | timeRequired | short description of how much time is required for method activity | text | methods |
 | category | a method's category name; do not capitalize | text | methods |
 
+## Managing dependencies
+
+This project uses Github's Dependabot to keep the NPM dependencies up to date.
+Dependabot takes care of noticing version updates and proposing the updates to
+`package.json` and `package-lock.json` as pull requests (PRs). A human
+developer needs to review these Dependabot PRs and merge them into the `main`
+branch.
+
+If Dependabot PRs go un-reviewed for too long, they can have merge conflicts
+and complex interactions with other code changes. The best practice is to
+review and merge Dependabot PRs as soon as possible. The automated test suite
+for this project is quite good, so this is fairly straightforward:
```

Comment:
Do we want to add some verbiage around semvar? e.g., it is recommended to test minor or major version changes manually? (rest LGTM)

Reply:
semvar major changes usually suggest that incompatible API changes were done, which ... could mean anything, i guess! 😓 which is why there's always an element of risk with major semvar changes. i'm not too sure how often this comes up though 🤔 

i'm fine merging as-is and if semvar issues do come up again we can re-clarify 🤷🏻‍♂️

Reply:
I added a paragraph suggesting additional testing might be appropriate.